rules/doctrine-code-quality/tests/Rector/Class_/RemoveRedundantDefaultAnnotationValuesRector/RemoveRedundantDefaultAnnotationValuesRectorTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\DoctrineCodeQuality\Tests\Rector\Class_\RemoveRedundantDefaultAnnotationValuesRector;
```

Comment:
Should tests for annotations for the whole class (like `@Entity`) be in the `Class_` namespace, or should they all be in `Property` as the rule is there?

Reply:
The namespace should be the same as the Rector rule has. So this is corret